db/migrate/20190910121338_add_wishlist_supported_tlds.rb

```diff
@@ -0,0 +1,15 @@
+class AddWishlistSupportedTlds < ActiveRecord::Migration[5.2]
+  def up
+    wishlist_supported_tld_description = <<~TEXT.squish
+      Supported TLD's for wishlist domain monitoring. First entry is used as a default autocomplete value. Separate with "|" (pipe) 
+    TEXT
+
+    wishlist_supported_tld_setting = Setting.new(code: :wishlist_supported_tld, value: 'ee|pri.ee|com.ee|med.ee|fie.ee',
```

Comment:
I think it would be better to use container here. It might be stored as JSON, if Entity–attribute–value model cannot be avoided. Parsing greatly increases complexity.